react/src/pages/ServingListPage.tsx

```diff
@@ -330,6 +334,15 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
                     <EndpointStatusTag endpointFrgmt={row} />
                   ),
                 },
+                {
+                  title: t('modelService.CreatedAt'),
+                  dataIndex: 'created_at',
+                  render: (created_at) => {
+                    return _humanReadableTime(created_at);
+                  },
+                  defaultSortOrder: 'descend',
```

Comment:
You can set it as ['descend', 'ascend', 'descend'] to prevent the sorter from reverting to its default status if you want. I think the default status of the sorter can be confusing.
```suggestion
                  defaultSortOrder: 'descend',
                  sortDirections: ['descend', 'ascend', 'descend'],
```